src/main/java/com/webcrawling/service/webCrawler/impl/WebCrawlerServiceImpl.java

```diff
@@ -41,6 +46,15 @@ public void  crawlWebsite(String startingUrl, int maxDepth, int maxDocuments) {
                     WebPage webPage = new WebPage();
                     webPage.setUrl(currentUrl);
                     webPage.setTitle(document.title());
+                    // Counting words and limiting to 200
+                    String content = document.text();
+                    String[] words = content.split("\\s+");
+                    int wordCount = words.length;
+
+                    if (wordCount > 200) {
+                        // Truncate the content to 200 words
+                        content = String.join(" ", Arrays.copyOfRange(words, 0, 200));
+                    }
```

Comment:
can you explain what are you doing here where are you setting content containing 200 character and why converting string to Array?